### ISSUE TITLE
knife doesn't save accross server rejoins/reboots

### ISSUE THREAD

**Issue by Wh1t3Rose** [kgns/weapons#93]
If you rejoin the server or the server reboots It gives you a random knife instead of the one that you were using last time.

However it does save the skin you picked for that knife.

Every column for each row in the weapons table (every gun and knife) has 0 as the value for each player.

**Comment by Wh1t3Rose** [comment]
![image](https://user-images.githubusercontent.com/4251482/59320135-36e04e00-8c9b-11e9-8a99-bc89154a58fb.png)

**Comment by Wh1t3Rose** [comment]
![image](https://user-images.githubusercontent.com/4251482/59320160-4a8bb480-8c9b-11e9-9586-6b1c9ec4132e.png)

**Comment by Wh1t3Rose** [comment]
@kgns

**Comment by Wh1t3Rose** [comment]
@kgns

**Comment by Wh1t3Rose** [comment]
Hello @kgns Can you help?

**Comment by kgns** [comment]
i cannot reproduce the problem with the info you have given

**Comment by Wh1t3Rose** [comment]
```
 SourceMod Version Information:
    SourceMod Version: 1.10.0.6415
    SourcePawn Engine: 1.10.0.6415, jit-x86 (build 1.10.0.6415)
    SourcePawn API: v1 = 5, v2 = 12
    Compiled on: May 15 2019 04:24:07
    Built from: https://github.com/alliedmodders/sourcemod/commit/f9ce25a
    Build ID: 6415:f9ce25a
    http://www.sourcemod.net/
```

``
[SM] Listing 66 plugins:
  01 "AFK Manager" (4.3.0) by Rothgar
  02 "CS:GO Movement Unlocker" (1.0) by Peace-Maker
  03 "Weapons & Knives" (1.3.3) by kgns | oyunhost.net
  04 "Hide Any Chat Triggers" (1.0) by Mr.Skullbeef
  05 "RNGFix" (1.1.2) by rio
  06 "Chat-Processor" (2.1.5) by Keith Warren (Drixevel)
  07 "SourceBans++: SourceComms" (1.6.3) by Alex, SourceBans++ Dev Team
  08 "Fun Votes" (1.10.0.6415) by AlliedModders LLC
  09 "SourceBans++ Report Plugin" (1.6.3) by RumbleFrog, SourceBans++ Dev Team
  10 "SourceBans++: Main Plugin" (1.6.3) by SourceBans Development Team, SourceBans++ Dev Team
  11 "Cronjobs" (2.0) by dordnung
  12 "SurfTimer" (2.2) by fluffys
  13 "Map Nominations Extended" (1.10.0) by Powerlord and AlliedModders LLC
  14 "Knife Drop / Pickup" (0.6) by SM9 (xCoderx)
  15 "VoiceAnnounceEx" (2.2.0) by Franc1sco franug, Mini and GoD-Tony
  16 "Custom Particles | Store Module" (1.0) by ban-hammer.net
  17 "CSGO Panorama Map Change Crashe Fixer" (1.0.6) by BOT Benson
  18 "CallAdmin" (0.1.8) by Impact, dordnung
  19 "csgotokens.com token changer" (1.2.1) by Adi - live4net
  20 "Fun Commands" (1.10.0.6415) by AlliedModders LLC
  21 "[Any]AutoKickAFK When ServerFull" (1.1.1) by Lux
  22 "Log Connections" (1.1) by Xander (Player 1)
  23 "Anti-Flood" (1.10.0.6415) by AlliedModders LLC
  24 "Nextmap" (1.10.0.6415) by AlliedModders LLC
  25 "Sound Commands" (1.10.0.6415) by AlliedModders LLC
  26 "SourceBans++: SourceSleuth" (1.6.3) by ecca, SourceBans++ Dev Team
  27 "[ANY] Chat Responder" (1.3.2) by Headline
  28 "Discord API" (0.1.97) by Deathknife
  29 "Admin File Reader" (1.10.0.6415) by AlliedModders LLC
  30 "Store" (1.2-dev) by Zephyrus
  31 "Basic Chat" (1.10.0.6415) by AlliedModders LLC
  32 "SourceBans++: Admin Config Loader" (1.6.3) by AlliedModders LLC, SourceBans++ Dev Team
  33 "Discord API" (1.0) by .#Zipcore, Credits: Shavit, bara, ImACow and Phire
  34 "Vote Mute/Vote Silence" (1.0.105) by <eVa>Dog/AlliedModders LLC
  35 "hConfigMenus" (1.1) by hAlexr
  36 "SM Voice Amount" (v1.4.1) by Franc1sco steam: franug
  37 "[ANY] Ban Disconnected Player" (2.0) by Headline, Original Plugin : mad_hamster
  38 "Discord: SourceComms" (1.1) by .#Zipcore
  39 "Basic Commands" (1.10.0.6415) by AlliedModders LLC
  40 "Rock The Vote Retry Fix" (1.8.0.5969) by AlliedModders LLC
  41 "Voice Hud" (1.7) by Agent Wesker
  42 "ConVar Enforcer" (1.0) by blacky
  43 "Basic Votes" (1.10.0.6415) by AlliedModders LLC
  44 "Dynamic" (0.0.32) by Neuro Toxin
  45 "SourceBans++: Bans Checker" (1.6.3) by psychonic, Ca$h Munny, SourceBans++ Dev Team
  46 "Discord: CallAdmin" (1.1) by .#Zipcore
  47 "[Any] Chat Logging Redux + RTF Support" (1.1.2) by McFlurry, Keith Warren (Shaders Allen)
  48 "Client Preferences" (1.10.0.6415) by AlliedModders LLC
  49 "Auto File Loader" (1.0.3) by Keith Warren (Shaders Allen)
  50 "SM Most Active" (2.6) by Franc1sco Steam: franug / shanapu
  51 "Anti-Micspam" (0.5) by FLOOR_MASTER and Bor3dGaming.com
  52 "Reserved Slots" (1.10.0.6415) by AlliedModders LLC
  53 "[CS:GO] Gun Menu" (1.2_fix) by LumiStance(adapting by Grey83)
  54 "Admin Menu" (1.10.0.6415) by AlliedModders LLC
  55 "MapChooser Extended" (1.10.2) by Powerlord, Zuko, and AlliedModders LLC
  56 "ServerAdvertisements3" (3.1.3) by ESK0
  57 "Basic Comm Control" (1.10.0.6415) by AlliedModders LLC
  58 "Player Commands" (1.10.0.6415) by AlliedModders LLC
  59 "Gloves" (1.0.3) by kgns | oyunhost.net
  60 "Admin Help" (1.10.0.6415) by AlliedModders LLC
  61 "Discord: SourceBans" (1.0) by .#Zipcore
  62 "Basic Info Triggers" (1.10.0.6415) by AlliedModders LLC
  63 "GameVoting" (1.8.8) by Neatek
  64 "Self-Mute Intelligence" (1.5.2) by IT-KiLLER and (Otokiru, edit 93x, Accelerator)
  65 "Thirdperson | Mirrow Mode" (1.11) by Zephyrus
  66 "Store - Trade System" (1.0) by Zephyrus
```

```
SM] Displaying 20 extensions:
[01] Automatic Updater (1.10.0.6393): Updates SourceMod gamedata files
[02] Webternet (1.10.0.6393): Extension for interacting with URLs
[03] Equinox Console Cleaner (1.8.0-manual): Bite me alien boi
[04] Accelerator (2.4.3): SRCDS Crash Handler
[05] CS Tools (1.10.0.6393): CS extended functionality
[06] BinTools (1.10.0.6393): Low-level C/C++ Calling API
[07] SDK Tools (1.10.0.6393): Source SDK Tools
[08] PTaH (P Tools and Hooks) (1.0.10): Expanding sdkhooks and sdktools
[09] SDK Hooks (1.10.0.6393): Source SDK Hooks
[10] DHooks (2.1.1): Dynamic Hooks
[11] Top Menus (1.10.0.6393): Creates sorted nested menus
[12] <OPTIONAL> file "system2.ext.so": /home/wh1t3rose/csgo_surf/csgo/addons/sourcemod/extensions/system2.ext.so: cannot open shared object file: No such file or directory
[13] Client Preferences (1.10.0.6393): Saves client preference settings
[14] MySQL-DBI (1.10.0.6393): MySQL driver implementation for DBI
[15] GeoIP (1.10.0.6393): Geographical IP information
[16] SMJansson (2.3.1/3): JSON parser/writer
[17] Regex (1.10.0.6393): Provides regex natives for plugins
[18] <OPTIONAL> file "store_sm.ext.so": /home/wh1t3rose/csgo_surf/csgo/addons/sourcemod/extensions/store_sm.ext.so: cannot open shared object file: No such file or directory
[19] SteamWorks Extension (1.2.3): Exposes SteamWorks functions to Developers
[20] SQLite (1.10.0.6393): SQLite Driver
```

@kgns

**Comment by kgns** [comment]
still cannot reproduce, is this still happening? are you sure you are using the official release and not a modified version?
